src/app/loadout-builder/process.ts

```diff
@@ -566,6 +545,39 @@ function getBaseStatValues(
   return statValues.map((statHash) => Math.max(baseStats[statHash], 0));
 }
 
+/**
+ * Get the stats totals attributed to locked mods. Note that these are stats from mods in a single bucket, head, arms, ect.
+ */
+function getLockedModStats(
+  lockedItems?: readonly LockedItemType[],
+  lockedArmor2Mods?: readonly LockedArmor2Mod[]
+): { [statHash: number]: number } {
+  const lockedModStats: { [statHash: number]: number } = {};
+  // Handle old armour mods
+  if (lockedItems) {
+    for (const lockedItem of lockedItems) {
+      if (lockedItem.type === 'mod') {
+        for (const stat of lockedItem.mod.investmentStats) {
+          lockedModStats[stat.statTypeHash] = lockedModStats[stat.statTypeHash] || 0;
+          lockedModStats[stat.statTypeHash] += stat.value;
```

Comment:
Not sure if this was moved from somewhere else, but this isn't really the right way to calculate the impact of mods, even though it happens to work for armor stats today. See https://github.com/DestinyItemManager/DIM/wiki/DIM's-New-Stat-Calculations. Might want to at least leave a note that this isn't accurate.

Reply:
Yeah this was just a copy paste into a function so I can reuse it. I will put a comment on it and fix it in another PR.